web/web/emails/send_emails.py

```diff
@@ -0,0 +1,32 @@
+import smtplib
+import web.emails.config as config
```

Comment:
This file is gitignored but you are requiring it here...don't ignore the file or don't import it here. If the config variables shouldn't be versioned controlled then get them from the environment and provide defaults.